package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kinvey-backbone-sdk",
   "version": "3.2.2",
```

Comment:
@thomasconner I'm assuming this will be updated before you release. It will jump to 3.4.1, right?